app/views/hub/organizations/_edit_organization.html.erb

```diff
@@ -1,5 +1,14 @@
 <%= f.cfa_input_field :name, t("general.name") %>
+<%= f.cfa_checkbox :coalition_id_nil, "This organization is not part of a coalition", options: {class: 'toggle-multi-select-state-visibility'} %>
```

Comment:
someone needs to set this to checked if the coalition id really is nil, which moves me in the direction of having a Form